wsdlgo/testdata/w3example1.golden

```diff
@@ -51,7 +51,7 @@ func (p *getEndorsingBoarderPortType) GetEndorsingBoarder(α *GetEndorsingBoarde
 			M GetEndorsingBoarderResponse `xml:"GetEndorsingBoarderResponse"`
 		}
 	}{}
-	if err = p.cli.RoundTripWithAction("GetEndorsingBoarder", α, &γ); err != nil {
+	if err = p.cli.RoundTripWithAction("http://www.snowboard-info.com/EndorsementSearch", α, &γ); err != nil {
```

Comment:
I think this should be `example.com`

Reply:
Fair point, doesn't make sense to have this domain here but I already merged. I'm happy to merge another PR to fix this. Sorry about that